code/game/turfs/simulated/minerals.dm

```diff
@@ -184,11 +177,10 @@
 /turf/simulated/mineral/random/high_chance/volcanic
 	environment_type = "basalt"
 	turf_type = /turf/simulated/floor/plating/asteroid/basalt/lava_land_surface
-	baseturf = /turf/simulated/floor/plating/lava/smooth/lava_land_surface
+	baseturf = /turf/simulated/floor/plating/asteroid/basalt/lava_land_surface
```

Comment:
right, so, I noticed that the baseturf of a lot of these is uh...lava? This is peculiar--not sure why it varies on TG versus here.

The only thing I can think of is that this is the ANCIENT version of lavaland where `ex_act(1)` or the singularity would remove it to the base turf, and the base turf WAS lava back (or was it chasms? I forget) then (that didn't last long and was too problematic)

Reply:
Alright, so after looking at this...yeah, going to have to revert this portion. 

Basically, they don't have the same baseturf system and they more gracefully handle turfs changing from one to another.

This means that you can RCD a turf or deconstruct things and it'll end up being lavaland's surface, but the singularity will still strip it down to the lava.

We don't have such a system where it acts as a list; it's one turf type and that's it, so changing this to the basalt versions is going to have unintended consequences as our system expects it to be the smooth lava and for us to use mapping helpers (TG doesn't use those anymore either).